network/p2p/fixture.go

```diff
@@ -0,0 +1,249 @@
+package p2p
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+	"time"
+
+	addrutil "github.com/libp2p/go-addr-util"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr/net"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	fcrypto "github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/metrics"
+	"github.com/onflow/flow-go/network/mocknetwork"
+	"github.com/onflow/flow-go/network/p2p/dns"
+	"github.com/onflow/flow-go/network/p2p/unicast"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// Workaround for https://github.com/stretchr/testify/pull/808
+const tickForAssertEventually = 100 * time.Millisecond
```

Comment:
maybe  rename to "ticksForAssertEventually"

Reply:
refactored in https://github.com/onflow/flow-go/pull/1643/commits/9ad1382715c16be64490f78a703a436cc607d00d